api/src/main/kotlin/com/gmtkgamejam/plugins/Auth.kt

```diff
@@ -32,10 +33,13 @@ fun Application.configureAuthRouting() {
 
                 val lifespanOfAppJwt = 86400000 // A user is logged into the Team Finder for 24 hours
 
+                // A securely random ID is used to ensure a JWT is unique, and that another JWT can't be brute-forced
```

Comment:
I dunno if this really helps the brute-forcing - the signing of a JWT is what prevents that. This should probably be some sort of user ID that can be correlated back to something useful (maybe Discord ID?), but it':s better than "some-Id" so ✅